shared/common/mixins.scss

```diff
@@ -25,6 +25,41 @@
   }
 }
 
+$breakpoints: (
+  xs: 320px,
+  sm: 768px,
+  md: 1024px,
+  lg: 1440px,
+  xl: 1920px,
+  xxl: 2560px,
+);
+
+@mixin breakpoint($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    $breakpoint-value: map-get($breakpoints, $breakpoint);
+    // Subtracts 0.5px to make breakpoint exclusive (width < x) instead of
+    // inclusive (width <= x)
+    // (Uses -0.5px instead of -1px due to incorrect behaviour on some
+    // systems ie. win10 with fractional display scaling ¯\_(ツ)_/¯)
+    @media (max-width: #{$breakpoint-value - 0.5px}) {
+      @content;
+    }
+  } @else {
+    @warn 'Invalid breakpoint: #{$breakpoint}.';
+  }
+}
+
+@mixin breakpointMobileFirst($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    $breakpoint-value: map-get($breakpoints, $breakpoint);
+    @media (min-width: #{$breakpoint-value}) {
+      @content;
+    }
+  } @else {
+    @warn 'Invalid breakpoint: #{$breakpoint}.';
+  }
+}
+
```

Comment:
To keep things simple I was going to just have a breakpoint for mobile (and maybe one for tablet size later if needed). I don't think we need all of them, since the ui layout works differently to the website and is already fairly flexible.

Reply:
U don't think it's better to have one breakpoint mixin (we can have only one or two items in the breakpoints list) and later we use the same mixin for mobile and tablet and maybe there will be need for some otehr breakpoint. 

I think this is better than to have isMobile and isTablet mixins. If you strongly think we should take this approach I can update all breakpoints in my code.

Reply:
Really either way works for me. I was just thinking that if we're probably only going to have a single breakpoint, then `isMobile` shorter to type and matches up with the hook name.

Reply:
But maybe it is better to future-proof it and use `breakpoint(mobile)`.

Reply:
Hm, do u think it's beneficial to also have mobileFirst breakpoint or no?  
I just feel a bit weird when have to write for example:
```
display: none;
isMobile() {...}
```

Probably because in the past I was always working with mobile first approach, but I'm fine if u think we should always take desktop first approach.

Reply:
I'm not sure if that makes things more confusing to have both types of breakpoints. I would probably stick to just the desktop first breakpoints.